_build/commands.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# this bash script generates stubs directories/index files based on the command JSONs
+
+COMMANDS="./_data/commands/latest/*.json"
```

Comment:
Where is the "/latest/" part coming from? From the valkey-doc repo?

Reply:
`/latest/` is just where I set the internal representation (it doesn't affect the URLs). I wanted the `site.data.commands` namespace in Jekyll to be a object not a map, so I needed the subdirectory. It could be anything, but `latest` made sense.